autospec/specfiles.py

```diff
@@ -1241,6 +1253,10 @@ def write_cmake_pattern(self):
 
         self._write_strip("\n")
         self.write_check()
+
+        if self.subdir:
+            self._write_strip("popd")
```

Comment:
The `popd` needs to be added before the `self.write_check()` two lines above, because that function will write a `%check` section if needed.

Reply:
So looking at the code, it adds ```cd clr-build; make test``` as its test code, in this case I think it needs the popd to be after the write_check.

Reply:
But the test code lives in a `%check` section, right? So that code is run in a different script...

Reply:
Ugh right, of course. That's a bit too ugly to support =/.

Reply:
Updated.